### ISSUE TITLE
vfs-http-adapter mountOptions.error?

### ISSUE THREAD

**Issue by orlin** [c9/vfs#25]
In [this module](https://github.com/c9/vfs/blob/master/http-adapter/restful.js), could we pass error options to the abort function through mountOptions.error for example.  The @isaacs [error-page module](https://npmjs.org/package/error-page) looks pretty good and would allow reuse of some custom errors.  Static vfs assets are the last thing in my [middleware stack](https://github.com/astrolet/astrolin/blob/active/web/app.coffee).  I rather avoid pre-checking if a file exists (through other middleware), i.e. hit the fs one extra time and bypass some of the more common vfs errors...

**Comment by creationix** [comment]
What if we made the abort function configurable?  Would that give you a flexible enough API?  I would inject the req and res objects so your could properly handle the error.

**Comment by orlin** [comment]
Perfect. Get the error and handle the `res` as one sees fit. This was about custom error pages, but redirecting (issue #26) would thus be covered as well.

**Comment by creationix** [comment]
Thanks.  Fixed in https://github.com/c9/vfs-http-adapter/commit/db036baa693354bf5b8c3c2b66b7d848f0800d62. Published as 0.2.3.

Note I moved vfs-http-adapter to it's own repo so it can get more attention. https://github.com/c9/vfs-http-adapter

To write a custom error handler, pass in a mountOption of `errorHandler`.

``` js
function errorHandler(req, res, err, code) {
  // Do your custom stuff here
}
```
